cypress/integration/display_faqs.js

```diff
@@ -0,0 +1,101 @@
+// @noflow
+/* global cy */
+
+describe('Display Static FAQ', () => {
+  it('User can navigate into FAQ Article', () => {
+    cy.visit('/');
+    cy.get('[data-cy=btn-nonexistent-booking]').trigger('click');
+    cy.get('.signInOrBack span').contains('Sign In');
+
+    cy
+      .get('[data-cy=faq-categories]')
+      .find('a')
+      .first()
+      .click({ force: true });
```

Comment:
Why is `force` necessary?

Reply:
Because height of the `a` tag is 0. So in theory its not visible

Reply:
This is interesting, if we give the `Link` a `display: block` attribute the 0 height doesn't happen and we don't have to "force" anything. 

This doesn't seem like a critical issue, but what do you think about this approach?